### ISSUE TITLE
Decoding an old Outlook Express address book has a "regular string array"

### ISSUE THREAD

**Issue by funwhilelost** [pboettch/libwab#3]
I was running an old Outlook Express book through the tool to save some old data and got this note in STDERR:

```
WARNING: Your wab file contains regular string array data.  I have NEVER seen
a file with this data type myself.  This will *attempt* to decode this data but
I don't know what will happen.  PLEASE check the results and send me an email
(--------) letting me know how things went.  Thanks
```

This is contact info for Sean Loring, the original dev, and I'm wondering if anyone is actively contributing to this utility and wants to see my example file.

From what I can tell that "string array" has a list of contact names in it because I'm getting a bunch of contacts listed with an email and no names in the STDOUT, but if I just look at the bytes in the WAB I can see the corresponding names _are_ in fact there.

**Comment by pboettch** [comment]
Yes, iirc, I simply imported the code I found in an old source-RPM file. Nothing was coded by me.

**Comment by funwhilelost** [comment]
I appreciate it, @pboettch - I may be able to contribute to the issue myself but it's low priority for me.  If anyone would like the example data I can share it.
